randomizer/Lists/CBLocations/DKIslesCBLocations.py

```diff
@@ -497,7 +497,7 @@
         name="Tree next to Japes",
         konglist=[Kongs.donkey, Kongs.diddy, Kongs.lanky, Kongs.tiny, Kongs.chunky],
         region=Regions.IslesMain,
-        logic=lambda l: l.climbing,
+        logic=lambda l: (l.climbing) or (Events.AirSpaceEntered in l.Events),
```

Comment:
needs l.isdiddy because if it's for any other kong, Diddy being able to fly doesn't mean the other kongs can pick it up